packages/backend/src/managers/apiServer.spec.ts

```diff
@@ -211,6 +211,19 @@ test('/api/tags returns ok', async () => {
   });
 });
 
+test('/api-docs/9000 returns swagger UI', async () => {
+  expect(server.getListener()).toBeDefined();
+  vi.mocked(modelsManager.getModelsInfo).mockRejectedValue({});
+  const listener = server.getListener();
+  if (!listener) {
```

Comment:
not related to this PR, but always feel weird than a **get** return something potentially undefined